tests/fixtures.py

```diff
@@ -627,6 +629,8 @@ def parametrize(params, values, **kwargs):
         ("mimikatz", "function=0x40105D", capa.features.common.String("ACR  > "), True),
         ("mimikatz", "function=0x40105D", capa.features.common.String("nope"), False),
         ("773290...", "function=0x140001140", capa.features.common.String(r"%s:\\OfficePackagesForWDAG"), True),
+        # overlapping string, see #1271
+        ("294b8d...", "function=0x404970,bb=0x404970,insn=0x40499F", capa.features.common.String("\r\n"), True),
```

Comment:
so we expect this string?

Reply:
correct. the sample contains the string "\r\n\r\n\x00HTTP" and this string is fetched using `&string[2]`. this test demonstrates the substring handling works as expected (and no embedded NULL bytes).

capa supports whitespace in strings, including newlines. filtering that out would be another discussion.

Reply:
Thanks for the clarification.
1. In `capa/features/extractors/strings.py` we do not extract `\r\n` so this seems inconsistent between file and instruction scope?
2. We have a lint to warn on strings with length smaller than 4, "capa only extracts strings with length >= 4"?

Should we make this uniform across the extractors or keep as is (since we have more knowledge at the instruction scope, for example)?

Reply:
ah, great points!

lets create two new issues, one for each of those points.

this is still the only test case i know of for this issue, but maybe we can find another that more clearly shows the problem.

Reply:
Maybe test here instead that the wrong string is not extracted?